allennlp/commands/elmo.py

```diff
@@ -268,34 +277,50 @@ def embed_file(self,
             The embeddings to output.  Must be one of "all", "top", or "average".
         batch_size : ``int``, optional, (default = 64)
             The number of sentences to process in ELMo at one time.
+        use_sentence_keys : ``bool``, optional, (default = False).
+            Whether or not to use full sentences as keys. By default,
+            the line numbers of the input file are used as ids, which is more robust.
```

Comment:
maybe add a brief explanation for why its more robust? e.g., something about the hdf5 spec disallowing `/` in key names?